google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessStateCollector.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Makes testing easier and also it has nothing to do with CloudDebugGlobalPoller's state, it's a completely independent functionality.